facebook-htb.js

```diff
@@ -116,7 +117,10 @@ function FacebookHtb(configs) {
         var queryObj = {
             sdk: '5.5.web',
             placementids: [],
-            adformats: []
+            adformats: [],
+            cachebuster: System.generateUniqueId(56, 'ALPHANUM'),
```

Comment:
I am surprised `'ALPHANUM'` works, I expected it to be a placeholder until I got the real API, because it's not in the stubs or the docs

Reply:
We have a enum defined where ALPHANUM is expected to be "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789" and generateUniqueId uses the charset to randomly generate the an ID. 

I believe this is in the README under https://github.com/indexexchange/facebook-adapter#-utility-libraries  - System but it's just not coded  in the stub library. Sorry if this caused some confusion.

Reply:
OK thanks, this is working correctly then.